js/app.js

```diff
@@ -94,44 +98,49 @@ app.controller('CheckoutController', function($scope, $interval, Order, $httpPar
           params: {'action': 'check_order', 'uuid': uuid},
           url: given_ajax_url
           }).then(function successCallback(response) {
-            if(response.data['status'] == "WAITING_FOR_DEPOSIT"){
-              $scope.order.altstatus = 0;
-            }
-            if(response.data['status'] == "DEPOSIT_RECEIVED"){
-              if(send_email == true){
-                var order_id = $scope.order.order_id;
-                var order_link = $scope.order.pagelink;
-                var order_coin = $scope.altcoinselect;
-                var order_coin_sym = $scope.order.altsymbol;
-                //Send Email
-                $http({
-                  method: 'GET',
-                  params: {'action': 'send_email', 'order_id':order_id, 'order_link':order_link, 'order_coin':order_coin, 'order_coin_sym':order_coin_sym},
-                  url: given_ajax_url
-                  });
-                send_email = false;
-              }
-              $scope.order.altstatus = 1;
-            }
-            if(response.data['status'] == "DEPOSIT_CONFIRMED"){
-              $scope.order.altstatus = 2;
-            }
-            if(response.data['status'] == "EXECUTED"){
-              $scope.order.altstatus = 3;
+            if(response.data['payment_status'] == "PAYMENT_RECEIVED" || response.data['payment_status'] == "PAYMENT_CONFIRMED"){
```

Comment:
The whole mutiple if conditional structure looks complex. Avoid duplicate code, use flags/switch case and simplify the code